wao/brain_config.py

```diff
@@ -0,0 +1,14 @@
+class BrainConfig:
+
+    BACKTEST_DUP = '0.45'
```

Comment:
same comment as above

@Bemhreth - this shows me that you did not review your pr